plugins/aztec/menu.sh

```
@@ -427,6 +432,14 @@ while true; do
       📦)
         sudo "${EDITOR}" "$PLUGIN_INSTALL_PATH"/docker-compose.yml
         ;;
+      🔒)
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 477.

(SC2221)